megamek/src/megamek/common/weapons/ATMHandler.java

```diff
@@ -129,7 +111,7 @@ protected int calcHits(Vector<Report> vPhaseReport) {
     @Override
     protected int calcAttackValue() {
         int av = 0;
-        int counterAV = 0;
+        int counterAV;
```

Comment:
Is this being initialized before use elsewhere in the code?

Reply:
I remove these on IDEA suggestion. Uninitialized ints are 0 anyway. I dont think this could possibly fail unless the compiler complains which it doesnt.

Reply:
If the compiler doesn't complain, I'm good. Hard to keep track of which languages default the value and which don't.